deepcell/data/tracking.py

```diff
@@ -53,7 +53,8 @@
 class Track(object):  # pylint: disable=useless-object-inheritance
 
     def __init__(self, path=None, tracked_data=None,
-                 appearance_dim=32, distance_threshold=64):
+                 appearance_dim=32, distance_threshold=64,
+                 **kwargs):
```

Comment:
Why are kwargs being exposed here? Is this coming from the other PR to expose crop mode and norm?

Reply:
Yeah I think that's a hold over from the other PR.